### ISSUE TITLE
kitty use viu blurred out

### ISSUE THREAD

**Issue by newyorkthink** [atanunq/viu#105]
I use kitty 
and play picture ,but is blurred out.

![image](https://user-images.githubusercontent.com/87791643/201571755-0ba41761-278d-4ba3-bddf-da8ebefba9f7.png)

**Comment by MTRNord** [comment]
I had the same issue. It seems that either kitty or zsh set $TERM to `xterm` while viu wants $TERM to be `xterm kitty` as seen in the demo in the readme.

However, it would be great if anyone knows why this is and how to fix it (properly). Especially as zsh says `zsh: can't find terminal definition for xterm kitty` when you set it manually.

**Comment by tshirtman** [comment]
> I had the same issue. It seems that either kitty or zsh set $TERM to `xterm` while viu wants $TERM to be `xterm kitty` as seen in the demo in the readme.
> 
> However, it would be great if anyone knows why this is and how to fix it (properly). Especially as zsh says `zsh: can't find terminal definition for xterm kitty` when you set it manually.

Indeed, using `env TERM="xterm kitty" viu path/to/image.png` worked for me, that’s at least a workaround if i can’t make that value default.

**Comment by newyorkthink** [comment]
> > I had the same issue. It seems that either kitty or zsh set $TERM to `xterm` while viu wants $TERM to be `xterm kitty` as seen in the demo in the readme.
> > However, it would be great if anyone knows why this is and how to fix it (properly). Especially as zsh says `zsh: can't find terminal definition for xterm kitty` when you set it manually.
> 
> Indeed, using `env TERM="xterm kitty" viu path/to/image.png` worked for me, that’s at least a workaround if i can’t make that value default.

It not about kitty , the problem is tmux

**Comment by MTRNord** [comment]
> > > I had the same issue. It seems that either kitty or zsh set $TERM to `xterm` while viu wants $TERM to be `xterm kitty` as seen in the demo in the readme.
> > > However, it would be great if anyone knows why this is and how to fix it (properly). Especially as zsh says `zsh: can't find terminal definition for xterm kitty` when you set it manually.
> > 
> > 
> > Indeed, using `env TERM="xterm kitty" viu path/to/image.png` worked for me, that’s at least a workaround if i can’t make that value default.
> 
> It not about kitty , the problem is tmux

The issue happens for me with zsh and kitty too. I dont use tmux at all.

**Comment by einarpersson** [comment]
Is it possible to get good quality with viu in tmux or is this impossible?
